src/Framework/ChangeWaves.cs

```diff
@@ -27,7 +27,8 @@ internal static class ChangeWaves
     {
         internal static readonly Version Wave17_10 = new Version(17, 10);
         internal static readonly Version Wave17_12 = new Version(17, 12);
-        internal static readonly Version[] AllWaves = { Wave17_10, Wave17_12 };
+        internal static readonly Version Wave17_13 = new Version(17, 13);
+        internal static readonly Version[] AllWaves = { Wave17_10, Wave17_12, Wave17_13 };
```

Comment:
So far, we've only done even changewaves. Would you mind updating the docs to say that? Or do you think we should change policy?

```suggestion
        internal static readonly Version Wave17_14 = new Version(17, 14);
        internal static readonly Version[] AllWaves = { Wave17_10, Wave17_12, Wave17_14 };
```